README.md

```diff
@@ -1,4 +1,31 @@
-# openFEC
+## Campaign finance for everyone
```

Comment:
Can we add something at the top of the documentation that says that this is the repo for the API? So you can know what you are looking at without reading the full project description.